doto-frontend/public/reminderWorker.js

```diff
@@ -0,0 +1,7 @@
+self.addEventListener("push", ev => {
+    const data = ev.data.json();
+    self.registration.showNotification(data.title, {
+        body: data.description,
```

Comment:
Currently, the notification just displays the task title and description, but the user might not know what to make of it.

We could modify the code here to display something like: 
REMINDER: \<task name\> due/happening at \<time\>

Might be a better UX.

Reply:
Yeah thats a good idea. I've added a commit to format the start date in the notification title.